src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+
+    private String name = "Rectangle";
```

Comment:
The variable 'name' should not be a class member since it's the same for all instances of the class. Instead, you can create a constant for it, or directly return the string in the 'getName' method.